src/main/java/com/oya/kr/community/controller/VoteController.java

```diff
@@ -0,0 +1,62 @@
+package com.oya.kr.community.controller;
+
+import java.security.Principal;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.oya.kr.community.service.VoteService;
+import com.oya.kr.global.dto.ApplicationResponse;
+import com.oya.kr.user.domain.User;
+import com.oya.kr.user.service.UserService;
+
+import lombok.RequiredArgsConstructor;
+
+/**
+ * @author 이상민
+ * @since 2024.02.18
+ */
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api/v1/votes")
```

Comment:
```suggestion
@RequestMapping("/api/v1/votes/{voteId}")
```
만약 해당 컨트롤러에 생성 관련된 로직을 두지 않을 거라면 위와 같은 path 가 반복될 것 같은데 분리하면 어떨까요??